examples/pagerduty-business-service/main.tf

```diff
@@ -1,15 +1,23 @@
+resource "random_id" "random_suffix" {
+  byte_length = 4
+}
+
+locals {
+  random_suffix = random_id.random_suffix.b64_url
+}
+
 module "mock_team" {
   source = "../../modules/pagerduty-team"
 
-  name        = "team - ${var.name}"
+  name        = "Team-${var.name}-${local.random_suffix}"
```

Comment:
capitalized on purpose?

Reply:
Yes

Reply:
Felt that team name deserved to be capitalized lol